packages/utilities/psammead-styles/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "7.0.4",
+  "version": "7.0.5",
```

```suggestion
  "version": "7.1.0",
```
I think changes like this should be minor bumps, I was explaining to Harry on this PR here: https://github.com/bbc/psammead/pull/4244#discussion_r559996973 

Sorry if a bit pedantic

---

Sorted